### DISCUSSION TITLE
Soy Dalto

### DISCUSSION THREAD

**Discussion by valeOopa** [soydaltogit/papa-dalto#3] [General]
Dalto jugatela y contestame esto porfa

**Comment by fe-row** [comment]
shh mewing

**Comment by aleksLlanos** [comment]
Dalto te voy a extrañar banda :((( Gracias por el contenido loco <3

**Comment by valeOopa** [comment]
Ayudaste a mucha gente con tus cursos y así va seguir siendo, espero que puedas lograr cualquier cosa que te propongas. Muchas gracias por tus cursos, sos lo mas grande que hay!!

**Comment by hdzMario** [comment]
Que le paso a Dalto ??

**Comment by Pablo-Barroso-Gonzalez** [comment]
Grande **Dalto** ayudas mucho con tus cursos gracias. ❤️
